app/views/users/team.html.haml

```diff
@@ -5,7 +5,7 @@
     .column-two-thirds
       %h1.heading-large= t('views.users.team.page_title')
     .column-one-third
-      = link_to t('views.users.new.page_title'), new_user_invitation_path, class: "button align-with-heading"
+      = link_to t('views.users.new.page_title'), new_user_invitation_path('advanced'), class: "button align-with-heading"
```

Comment:
Do we need to pass the param into the link_to method i.e. `new_user_invitation_path(role: 'advanced')`